Source/Core/DolphinQt/Config/Graphics/AdvancedWidget.cpp

```diff
@@ -164,6 +190,17 @@ void AdvancedWidget::ConnectWidgets()
   connect(m_load_custom_textures, &QCheckBox::toggled, this, &AdvancedWidget::SaveSettings);
   connect(m_dump_use_ffv1, &QCheckBox::toggled, this, &AdvancedWidget::SaveSettings);
   connect(m_enable_prog_scan, &QCheckBox::toggled, this, &AdvancedWidget::SaveSettings);
+  connect(m_scaled_efb_exclude_enable, &QCheckBox::toggled, [=](bool checked) {
+    m_scaled_efb_exclude_slider->setEnabled(checked);
+    m_scaled_efb_exclude_label->setEnabled(checked);
+  });
+
+  // A &QSlider signal won't fire when game ini's trigger a change, due to a signalblock in
+  // GraphicsSlider
+  connect(m_scaled_efb_exclude_slider, &GraphicsSlider::valueChanged, [=] {
+    m_scaled_efb_exclude_label->setText(tr("Size  <  ") +
```

Comment:
Instead of 'Size', 'Width' would be more accurate?

Reply:
Also this can be

`m_scaled_efb_exclude_label->setText(tr("Size < %1").arg(m_scaled_efb_exclude_slider->value())`

Reply:
Agree, thanks, implemented this here and in the LoadSettings function a few lines down. Based on the comment here looks like it was duplicated on purpose due to "A &QSlider signal won't fire when game ini's trigger a change, due to a signalblock in GraphicsSlider".